frontend/app/components/wp-create/wp-create.controller.ts

```diff
@@ -97,7 +99,15 @@ export class WorkPackageCreateController {
             });
         }
       })
-      .catch(error => this.wpNotificationsService.handleErrorResponse(error));
+      .catch(error => {
+        if (error.errorIdentifier == "urn:openproject-org:api:v3:errors:MissingPermission") {
+          let url: string = $location.absUrl();
+          $location.path('/login').search({back_url: url});
+          let loginUrl: string = $location.absUrl();
+          window.location.href = loginUrl;
```

Comment:
I believe you should double check to make sure that the user is actually not logged in and not only lacking permissions despite being logged in. If it is the later, redirecting to the login page will cause confusion. 

I think that this scenario is causing the spec failure you reported.

To double check, you'd have to fetch the root resource (/api/v3) which will have a user link if the user is logged in.